src/Settings/Setting.js

```diff
@@ -87,6 +92,24 @@ function settingForceOneColumn () {
   }
 }
 
+function settingDarkMode () {
+  const darkMode = State.metadata.get('darkMode')
+  if (settings.darkMode !== darkMode) {
+    State.metadata.set('darkMode', settings.darkMode)
+  }
+
+  if (settings.darkMode) {
+    jQuery('html').addClass('dark')
+  } else {
+    jQuery('html').removeClass('dark')
+  }
```

Comment:
I think it's better to use classes on the body rather than the html tag.

Reply:
Apparently not! Stateful things that involve classes should apparently not be attached to <body>. source: TME on the Twine discord

Reply:
Huh, ok.